rootfs/Dockerfile

```diff
@@ -1,4 +1,14 @@
-FROM alpine:3.1
+FROM alpine:3.3
+
+# install git and configure gituser
+ENV GITHOME /home/git
+ENV GITUSER git
+# this is so the minio client (https://github.com/minio/mc) works properly
+ENV DOCKERIMAGE=1
+ENV DEIS_RELEASE 2.0.0-dev
```

Comment:
What if no changed occurred that release? The version bump would be a no-op and the cached layers would stay the same, which would make the diff between two versions minimal. What would be the reason behind busting the cache in this situation?